docs/admin.md

```diff
@@ -0,0 +1,88 @@
+| db-service.yaml       | corev1.Service      | For DB enabled| PostgreSQL Service   |
+| db-secret.yaml        | corev1.Secret       | For DB enabled| Secret to connect Backstage to PSQL   |
+| route.yaml            | openshift.Route     | No (for OCP)  | Route exposing Backstage service    |
+| app-config.yaml       | corev1.ConfigMap    | No            | Backstage app-config.yaml    |
+| configmap-files.yaml  | corev1.ConfigMap    | No            | Backstage config file inclusions from configMap   |
+| configmap-envs.yaml   | corev1.ConfigMap    | No            | Backstage env variables from configMap    |
+| secret-files.yaml     | corev1.Secret       | No            | Backstage config file inclusions from Secret   |
+| secret-envs.yaml      | corev1.Secret       | No            | Backstage env variables from Secret    |
+| dynamic-plugins.yaml  | corev1.ConfigMap    | No            | dynamc-plugins config *    |
+
+
+NOTES: 
+ - Mandatory means it is needed to be present in either (or both) Default and CR Raw Configuration.
+ - dynamic-plugins.yaml is a fragment of app-config.yaml provided with RHDH/Janus-IDP. The reason it mentioned separately is specific way it provided (via mounting to dedicated initContainer)  
+
+### Operator Bundle configuration 
+
+With Backstage Operator's Makefile you can generate bundle descriptor using *make bundle* command
+
+Along with CSV manifest it generates default-config ConfigMap manifest, which can be modified and applied to Backstage Operator
+TODO: is kubectl sufficient or there are some other tools?
+
+### Kustomize deploy configuration
+
+Make sure use the current context in your kubeconfig file is pointed to correct place, change necessary part of your config/manager/default-config or just replace some of the file(s) with yours and run
+``
+make deploy
+``
+
+### Direct ConfigMap configuration
+
+You can change default configuration by directly changing the default-config ConfigMap with kubectl like:
+
+ - if you need to get existed default-config
+
+``
+kubectl get -n backstage-system configmap default-config > my-config.yaml
+``
+
+- modify any part of it (yes, it could be fairly big file)
+
+- apply it back
+
+``
+  kubectl apply -n backstage-system -f my-config.yaml
+``
+
+It has to be re-applied to the controller's container after some time.
+
+
+### Use Cases
+
+#### Airgapped environment
+
+Creating Backtage CR, the Operator will try to create Backstage Pod, deploying:
+- Backstage Container from the image, configured in *(deployment.yaml).spec.template.spec.Containers[].image*
+- Init Container (applied for RHDH/Janus-IDP configuration, usually the same as Backstage Container)
+- 
+Also, if Backstage CR configured as EnabledLocalDb it will create Database (PGSQL) Container, configured in *(db-deployment.yaml).spec.template.spec.Containers[].image*
+
+By default Backstage Operator is configured to use publicly availabe images, which is not acceptable for Airgapped environment.
```

Comment:
"not acceptable for Airgapped environment"

This is an unacceptable solution. We can't just say "if you want postgres, don't use airgap".

Instead we need a solution that will prefetch the relevant container inside the cluster.... https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.10/html-single/administration_guide/index#installing-devspaces-in-a-restricted-environment ... and then use that same image and pinned digest when deploying the postgres pod.

Reply:
see also https://github.com/janus-idp/operator/pull/86 which seeks to provide and OOTB airgap solution

Reply:
I just noticed this unresolved comment. Can we rephrase this, e.g. to highlight the `RELATED_IMAGE_*` environment variables?

Reply:
I do not think I tried to say "if you want postgres, don't use airgap", it is just stated that default config uses publicly available images (which does not seem to be a problem?)

As discussed this topic is not finished and has to be carefully [tested and documented](https://github.com/janus-idp/operator/issues/80#issuecomment-1864733601) .

Reply:
suggested some improved copy in https://github.com/janus-idp/operator/pull/64#discussion_r1439691147